### ISSUE TITLE
Squiz.WhiteSpace.FunctionOpeningBraceSpace.SpacingAfter for closures?

### ISSUE THREAD

**Issue by VasekPurchart** [squizlabs/PHP_CodeSniffer#413]
This example is not reported in the 1.5.x versions and in the 2.x it is, is this an intended behavior, that closures are now checked as well?
``` php
$foo = function () {

	foo();
};
```
Since normal functions/methods and closures are rather different (in the context that they are used) I would vote against using this for closures (I have found some examples in our codebase, where the lines actually improve readability).

**Comment by gsherwood** [comment]
Yes, this is intentional in the Squiz standard. We treat functions and closures the same way.

You'll need to create a custom coding standard if you want different rules for them, although I don't have a sniff that enforces different formatting for closures, so you'd probably need to write one.

**Comment by VasekPurchart** [comment]
Ok, thanks for clarification, just wanted do know, if it was an intended change.
